h2/utilities.py

```diff
@@ -147,9 +148,17 @@ def authority_from_headers(headers):
     return None
 
 
-def validate_headers(headers):
+# Flags used by the validate_headers pipeline to determine which checks
+# should be applied to a given set of headers.
+HeaderValidationFlags = collections.namedtuple('HeaderValidationFlags',
+                                               ['is_client', 'is_trailer'])
```

Comment:
Lets do some indentation cleanups here. Rather than constrain ourselves to only having a tiny number of characters to work with, let's reflow to:

```python
HeaderValidationFlags = collections.namedtuple(
    'HeaderValidationFlags',
    ['is_client', 'is_trailer']
)
```